data/korath/korath news.txt

```diff
@@ -0,0 +1,72 @@
+		word
+			"Korath merchant"
+			"Kor Efret merchant"
+			"Korath vendor"
+			"Kor Efret vendor"
+	message
+		word
+			"A Korath waves you over to a stand filled with a variety of foods. It wants you to buy something, but nothing looks particularly appetizing."
+			"You spend some time inspecting the wares of a street vendor selling small pieces of... some sort of technology. You have no idea what any of it is."
+			"A merchant cries out in its own language to passerby while gesturing to a strange outfit you've never seen before. Some stop to look, but most ignore it."
+			"One Korath seems to be auctioning off a strange device to a small crowd."
+			"You enter an unusually busy building and are greeted by a Korath who seems to work there. It's filled with monitors displaying natural flora and fauna. The Korath in here are captivated by them."
+
+news "kor efret worker"
+	location
+		government "Kor Efret"
+	name
+		word
+			"Korath worker"
+			"Kor Efret worker"
+	message
+		word
+			"The Korath attaching fuel cables to your ship stops and holds both its hands up with its palms toward you when it notices you watching."
+			"Two of the workers nearby are speaking to each other while gesturing to your ship. It looks like they aren't used to visitors here."
+			"A worker carries a crate containing several tools past you to several other Korath. They seem to be repairing an outfit - or maybe it's an air conditioning unit. You can't really tell."
+			"A Korath emerges from a nearby dilapidated building and begins sealing every possible entrance - windows, doors, large cracks - shut. It moves with such efficiency that you're sure it's done this plenty of times before."
+
+news "kor efret local"
+	location
+		government "Kor Efret"
+	name
+		word
+			"Korath local"
+			"Kor Efret local"
+			"Korath"
+			"Kor Efret"
+	message
+		word
+			"A ball chased by a young Korath rolls up to your feet. You pick it up, and the youth pauses and stares at you for a moment before gesturing in thanks and accepting the ball. It runs back to a grown Korath who holds both its hands up with palms toward you."
+			"There's a Korath nearby precariously balancing several boxes above its head. You can't tell if it's a street performer or just transporting the crates."
+			"A local Korath gestures toward a building you've never been inside before. Out of curiousity, you poke your head inside to see the reptilian aliens conversing and sharing drinks: it's a bar. Some things never change, no matter where you are in the galaxy."
+			"A Kor Efret near your ship is speaking with a worker, gradually increasing its volume until it's doing some strange mix of shouting and hissing. Suddenly, it turns and stalks away with its chest puffed out."
+			"You try asking one of the Korath about the abandoned buildings littered throughout the spaceport, but it's no good. You're unable to convey the strange dichotomy the mixture of old and new makes; in fact, it seems to be the norm here."
```

Comment:
Not sure what this one is trying to say. Given that the player can't communicate with the Korath, I'd just remove it.